ansible/library/switch_capabilities_facts.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 
-import swsssdk
+from swsscommon import swsscommon
```

Comment:
>swsscommon [](http://example.com/codeflow?start=23&length=10)

Is it compatible to old images?

Reply:
The code change will break on old device, I will update code and test with old image on by devbox.

Reply:
The code break on 201811 branch because can't find swsscommon, will improve code to fix this issue:
admin@vlab-01:~$ python ./acl_capabilities_facts.py
Traceback (most recent call last):
  File "./acl_capabilities_facts.py", line 3, in <module>
    from swsscommon import swsscommon
ImportError: No module named swsscommon

Reply:
The sonic-mgmt master branch will not be used to test 201811 and older branch images. If this is the truth, then it is good enough, and no need to improve.

Reply:
The fix is necessary because SonicDBConfig.load_sonic_global_db_config() added on 2021, so on 2019 swsscommon exist but we still need use swsssdk.